content/faq/app/which-ios-and-android-versions-are-supported/index.es.md

```diff
@@ -0,0 +1,16 @@
+---
+title: ¿Qué versiones de iOS y Android están soportadas?
+description: "Preguntas frecuentes de la aplicación Organic Maps"
```

Comment:
Don't you want to add a localized slug for better SEO? E.g.
`slug: que-versiones-de-ios-y-android-estan-soportadas`

Reply:
sluggized!!